src/store.ts

```diff
@@ -111,6 +111,13 @@ interface WrapperSDKInfo {
     isInfoSet: boolean;
 }
 
+export interface IFeatureFlags {
+    reportBatching?: string;
+    eventBatchingIntervalMillis?: string;
+    offlineStorage?: string;
+    directURLRouting?: boolean;
+}
```

Comment:
As per our conversation on Zoom, this should be a number. All types used within the SDK should conform to JS, even if the server returns only strings.

Reply:
Create a TODO for `offlineStorage`